<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnnouncementSettingWidget</class>
 <widget class="QWidget" name="AnnouncementSettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>554</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBoxEffects">
     <property name="title">
      <string>Effects</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBoxUseFading">
        <property name="toolTip">
         <string>If checked, when switching displayed text, fades the old text out and fades the new text in .</string>
        </property>
        <property name="text">
         <string>Use fading effects</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxUseShadow">
        <property name="toolTip">
         <string>Useful when using a wallpaper image. Displays a fancy shadow effect.</string>
        </property>
        <property name="text">
         <string>Use shadow</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxUseBlurredShadow">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Useful when using a wallpaper image. Displays a fancy shadow effect.</string>
        </property>
        <property name="text">
         <string>Use blurred shadow</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxBackground">
     <property name="title">
      <string>Use Background Image</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineEditBackground">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonBackground">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonApplyToAll">
        <property name="toolTip">
         <string>Apply this background to all active backgrounds.</string>
        </property>
        <property name="icon">
         <iconset resource="softprojector.qrc">
          <normaloff>:/icons/icons/common.png</normaloff>:/icons/icons/common.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxTextProperties">
     <property name="title">
      <string>Text Properties</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QToolButton" name="toolButtonColor">
          <property name="text">
           <string>Color...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicViewTextColor">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonFont">
          <property name="text">
           <string>Font...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelFont">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="labelAlignment">
          <property name="text">
           <string>Alignment:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxVerticalAling">
          <item>
           <property name="text">
            <string>Top</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Middle</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bottom</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxHorizontalAling">
          <item>
           <property name="text">
            <string>Left</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Center</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Right</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxUseDisp2">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>0</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>0</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>0</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>170</red>
           <green>85</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>0</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>0</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>0</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>170</red>
           <green>85</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>153</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="title">
      <string>Use Separate Secondary Display Screen Settings</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBoxEffects2">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>0</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>0</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="title">
         <string>Effects</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QCheckBox" name="checkBoxUseFading2">
           <property name="toolTip">
            <string>If checked, when switching displayed text, fades the old text out and fades the new text in .</string>
           </property>
           <property name="text">
            <string>Use fading effects</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxUseShadow2">
           <property name="toolTip">
            <string>Useful when using a wallpaper image. Displays a fancy shadow effect.</string>
           </property>
           <property name="text">
            <string>Use shadow</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxUseBlurredShadow2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Useful when using a wallpaper image. Displays a fancy shadow effect.</string>
           </property>
           <property name="text">
            <string>Use blurred shadow</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxBackground2">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>0</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>0</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="title">
         <string>Use Background Image</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="lineEditBackground2">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonBackground2">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxTextProperties2">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>0</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>0</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="title">
         <string>Text Properties</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QToolButton" name="toolButtonColor2">
             <property name="text">
              <string>Color...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGraphicsView" name="graphicViewTextColor2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonFont2">
             <property name="text">
              <string>Font...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelFont2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="labelAlignmentl2">
             <property name="text">
              <string>Alignment:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxVerticalAling2">
             <item>
              <property name="text">
               <string>Top</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Middle</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Bottom</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxHorizontalAling2">
             <item>
              <property name="text">
               <string>Left</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Center</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Right</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer15">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>59</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonDefault">
       <property name="text">
        <string>Reset All To Default</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBoxUseFading</tabstop>
  <tabstop>checkBoxUseShadow</tabstop>
  <tabstop>checkBoxUseBlurredShadow</tabstop>
  <tabstop>groupBoxBackground</tabstop>
  <tabstop>buttonBackground</tabstop>
  <tabstop>lineEditBackground</tabstop>
  <tabstop>toolButtonColor</tabstop>
  <tabstop>graphicViewTextColor</tabstop>
  <tabstop>toolButtonFont</tabstop>
  <tabstop>comboBoxVerticalAling</tabstop>
  <tabstop>comboBoxHorizontalAling</tabstop>
  <tabstop>groupBoxUseDisp2</tabstop>
  <tabstop>checkBoxUseFading2</tabstop>
  <tabstop>checkBoxUseShadow2</tabstop>
  <tabstop>checkBoxUseBlurredShadow2</tabstop>
  <tabstop>groupBoxBackground2</tabstop>
  <tabstop>buttonBackground2</tabstop>
  <tabstop>lineEditBackground2</tabstop>
  <tabstop>toolButtonColor2</tabstop>
  <tabstop>graphicViewTextColor2</tabstop>
  <tabstop>toolButtonFont2</tabstop>
  <tabstop>comboBoxVerticalAling2</tabstop>
  <tabstop>comboBoxHorizontalAling2</tabstop>
  <tabstop>pushButtonDefault</tabstop>
 </tabstops>
 <resources>
  <include location="softprojector.qrc"/>
 </resources>
 <connections/>
</ui>
